tools/lib/helpers.py

```diff
@@ -9,7 +9,7 @@ class RE:
 
   INDEX = r'-?[0-9]+'
   SLICE = fr'(?P<start>{INDEX})?:?(?P<end>{INDEX})?:?(?P<step>{INDEX})?'
-  SEGMENT_RANGE = fr'{ROUTE_NAME}(?:(--|/)(?P<slice>({SLICE})))?(?:/(?P<selector>([qras])))?'
+  SEGMENT_RANGE = fr'{ROUTE_NAME}(?:(--|/)(?P<slice>({SLICE})))?(?:/(?P<selector>([qra])))?'
```

Comment:
make sure to update the dataset

Reply:
already done

Reply:
I don't have permission to push to the hugging face